lib/modules/dosomething/dosomething_user/dosomething_user.install

```diff
@@ -143,3 +143,19 @@ function dosomething_user_update_7012() {
   field_delete_field('field_school_id');
   field_delete_field('field_user_registration_source');
 }
+
+/**
+ * Remove blocked Ips table that isn't used and remove IPs from sessions table.
+ */
+function dosomething_user_update_7013()
+{
+  $tables = 'blocked_ips';
```

Comment:
Is this a Drupal core thing? It might be safest to just truncate for now.

Reply:
it's actually empty right now. I can just remove this. There is nothing in it on prod